trie/zk/tree_path.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package zk
+
+import (
+	zkt "github.com/kroma-network/zktrie/types"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// TreePath 0 left, 1 right
+type TreePath []byte
+
+var (
```

Comment:
Couldn't we share the `left` and `righ` in `iterator.go`?

```suggestion
const (
```